src/pages/on-chain-governance/index.page.tsx

```diff
@@ -253,9 +273,9 @@ function UserQueryButtonRow({
           <a
             data-testid="OnChainGovernance.AllProposalsButton"
             className={classNames(
-              "md:font-normal font-medium rounded-l border border-r-0 py-[6px] md:px-[25px] px-3 md:text-base text-xs text-gray-900 border-gray-200",
+              "md:font-normal font-medium rounded-l border border-r-0 py-[6px] md:px-[25px] px-3 md:text-base text-xs text-gray-900 dark:text-dark-gray-900 border-gray-200 dark:border-dark-gray-200",
```

Comment:
Check bg for buttons for darkmode 
<img width="190" alt="image" src="https://user-images.githubusercontent.com/57183851/212609760-9f93cb29-e6ff-4606-8dd0-b54c8dc33694.png">

<img width="344" alt="image" src="https://user-images.githubusercontent.com/57183851/212609847-cacb54bc-02c6-4aa5-aeae-630882b8937f.png">